CONTRIBUTING.md

```diff
@@ -89,16 +99,30 @@ There are examples of such models in sa/model.go, along with code to
 turn a model into a `struct` used internally. 
 
 An example of a flag-gated migration, adding a new `IsWizard` field to Person
-controlled by a `allowWizards` flag:
+controlled by a `AllowWizards` feature flag:
 
 ```
+# features/features.go:
+
+const (
+	unused FeatureFlag = iota // unused is used for testing
+	AllowWizards // Added!
+)
+
+...
+
+var features = map[FeatureFlag]bool{
+	unused: false,
+	AllowWizards: false, // Added!
```

Comment:
There is no way to automatically enumerate the existing `FeatureFlag` consts and add them to a map so it has to be done by hand, it's not related to the zero value.